chore: Add `CONTRIBUTING.md` and update `README.md` (#64)
momentohq/client-sdk-python: README.md

# Momento client-sdk-python

> Python SDK for Momento, which allows developers to easily use cache that works at scale without any of the operational overhead required by traditional caching solution.

Reviewer: ```suggestion
Python SDK for Momento, a serverless cache that automatically scales without any of the operational overhead required by traditional caching solutions.
```

<br/>

## Getting Started :running:

### Requirements

- [Python 3.7](https://www.python.org/downloads/) or above is required
- A Momento Auth Token is required, you can generate one using the [Momento CLI](https://github.com/momentohq/momento-cli)

<br/>

### Installing Momento and Running the Example

Check out our [Python SDK example repo](https://github.com/momentohq/client-sdk-examples/tree/main/python)!

<br/>

### Using Momento

```python
import os
import momento.simple_cache_client as simple_cache_client

# Initializing Momento
_MOMENTO_AUTH_TOKEN = os.getenv('MOMENTO_AUTH_TOKEN')
_ITEM_DEFAULT_TTL_SECONDS = 60
simple_cache_client.init(_MOMENTO_AUTH_TOKEN, _ITEM_DEFAULT_TTL_SECONDS) as cache_client

# Creating a cache named "cache"
_CACHE_NAME = 'cache'
cache_client.create_cache(_CACHE_NAME)

# Sets key with default TTL and get value with that key
_KEY = 'MyKey'
_VALUE = 'MyValue'
cache_client.set(_CACHE_NAME, _KEY, _VALUE)
get_resp = cache_client.get(_CACHE_NAME, _KEY)

> print(f'Looked up Value: {str(get_resp.value())}')\

Reviewer: extra `\` at the end?
